src/about.c

```diff
@@ -87,7 +87,7 @@ static const gchar *translators[][2] = {
 	{ "sv", "Tony Mattsson &lt;superxorn@gmail.com&gt;" },
 	{ "sr", "Nikola Radovanovic &lt;cobisimo@gmail.com&gt;"},
 	{ "tr", "Gürkan Gür &lt;seqizz@gmail.com&gt;"},
-	{ "uk", "Boris Dibrov &lt;dibrov.bor@gmail.com&gt;" },
```

Comment:
I'd keep Boris in the list -- but the list somehow is outdated anyway. We have to find a better way than a staitc list :/

Reply:
I don't mind, in general. However you can also notice that [Boris' name __is__ mentioned in the list](https://github.com/geany/geany/blob/7b7f0def021715a44ac1e8c3938becb28270b71f/src/about.c#L102) under the "Previous Translators" heading.

Since there seems to have not been any updates to 'uk.po' by Boris since 2008, I assumed he indeed is no longer handling this translation and thus I moved his name to the "Previous Translators" list (it's visible in `About > Credits`).

Mainly it's the apparent lack of attention to this translation is what prompted me to very much overhaul it from scratch. Well, I'm using Geany with the Ukrainian UI, so I'm naturally interested in maintaining its usability and consistency.

Reply:
Well.... true. I will merge this and we need to rethink the about.c in another place